### ISSUE TITLE
Make this repository mybinder.org friendly

### ISSUE THREAD

**Issue by rgaiacs** [gesiscss/WebBot-tutorials#3]
Can we add an `environment.yaml` file? More details at https://repo2docker.readthedocs.io/en/latest/config_files.html#environment-yml-install-a-conda-environment.

```
Executing 'example.ipynb'
  Cell 1/11: ''...

An error occurred while executing the following cell:
------------------
from webbotparser import WebBotParser
parser = WebBotParser(engine = 'DuckDuckGo News')
------------------


---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 1
----> 1 from webbotparser import WebBotParser
      2 parser = WebBotParser(engine = 'DuckDuckGo News')

ModuleNotFoundError: No module named 'webbotparser'
```

**Comment by yfiua** [comment]
How can i test it?

**Comment by rgaiacs** [comment]
1. Visit https://mybinder.org/
2. Provide the URL of this GitHub repository
3. Provide the name of the feature branch
4. Click in `launch`

**Comment by yfiua** [comment]
it's fully reproducible on mybinder now

**Comment by rgaiacs** [comment]
Thanks!

**Comment by rgaiacs** [comment]
Where is the `environment.yaml`?

I'm getting

```
An error occurred while executing the following cell:
------------------
import WebSearcher as ws
import json
import pandas as pd
------------------


---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[2], line 3
      1 import WebSearcher as ws
      2 import json
----> 3 import pandas as pd

ModuleNotFoundError: No module named 'pandas'
```

**Comment by yfiua** [comment]
Fixed in [0ef2566](https://github.com/gesiscss/WebBot-tutorials/commit/0ef2566608788d5b449f2e07593c2a17d9622c52).
